### ISSUE TITLE
Silverstripe 3?

### ISSUE THREAD

**Issue by srmcatee** [nyeholt/silverstripe-advancedreports#4]
Hi.  I really like your module.  And have incorporated it into several sites.  

I noticed when upgrading to Silverstripe 3 much of the module does not work.

Here are a few of the problems I've found so far.
- Reference to ModelAdmin_RecordController which is not present in ss3.
- FieldSet is not deprecated to FieldList.
- Also, the submit button does not appear to be showing.  I have not fixed this one yet.  

Do you have any plans on releasing a fixed ss3 compliant version of the module?

Thanks for the time you put into this.  It really helped a few sites I built.

Steve

**Comment by nyeholt** [comment]
Yes, there's been several things updated just recently :). There's still a few more to go, but for the most part it's up and working.

**Comment by spekulatius** [comment]
@srmcatee @lawrencehowlett I have been working on getting the module further to be stable for SS 3.1. @nyeholt merge the pull requests. Feel free to open new issues at my fork and I'll see if I can fix them.
